BraintreeCore/src/main/java/com/braintreepayments/api/ExperimentalBetaApi.kt

```diff
@@ -0,0 +1,4 @@
+package com.braintreepayments.api
+
+@RequiresOptIn
+annotation class ExperimentalBetaApi
```

Comment:
Can this annotation include docstrings? Will they be generated/published  like any other class? If so, let's explain what ExperimentalBetaApi means in our SDK in the doc strings

Reply:
@sarahkoop please take a look at the language.

Reply:
Maybe let's use the same comment that's currently on the beta classes:

`The annotated method or class is in beta. It's public API may change or be removed in future releases.`